### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by leoj3n**
I don't remember when/where we talked about this, but I just wanted to open a PR so it doesn't get lost/forgotten. I think @matthewp had some reservations about updating the README, and mentioned something about list of plugins that the org maintains.

### CODE REVIEW

**Comment by matthewp** [comment]
I'm ok with updating the readme. I'll review this.

File: README.md [hunk 1]
```diff
@@ -7,14 +7,105 @@
 
 [![Sauce Test Status](https://saucelabs.com/browser-matrix/matthewphillips.svg)](https://saucelabs.com/u/matthewphillips)
 
-`steal` is a JavaScript module loader. `steal` supports ES6, AMD, CommonJS, LESS, and 
-CSS modules. It makes development workflows simple.
+In addition to a collection of plugins, __StealJS__ is comprised of two main components:
 
-It is one half of the [StealJS](http://stealjs.com) project.  It's other half,
-[steal-tools](https://github.com/stealjs/steal-tools) provides high performance builds.
+  - **Steal**: an extensible, universal module loader.
```
Comment (matthewp) [inline README.md]: we use lower-case `steal`

File: README.md [hunk 2]
```diff
@@ -7,14 +7,105 @@
...
+  - **Steal Tools**: utilities for building, transforming, and exporting module formats.
```
Comment (matthewp) [inline README.md]: Lower case `steal-tools`

File: README.md [hunk 3]
```diff
@@ -7,14 +7,105 @@
...
+  - [Less](https://github.com/stealjs/steal-less)
+  - [Conditional](https://github.com/stealjs/steal-conditional)
+  - [Mocha](https://github.com/stealjs/steal-mocha)
+  - [QUnit](https://github.com/stealjs/steal-qunit)
+
+For example, the Less plugin allows Less files to be loaded similarly to JavaScript modules:
+
+```js
+// ES6
+import "style.less!";
```
Comment (matthewp) [inline README.md]: The bang is not necessary so we omit it now.

**Comment by matthewp** [comment]
Just a couple of things, thanks!

**Comment by leoj3n** [comment]
@matthewp Did you also comb through the code examples? I took my best stab at it, but am not entirely sure I hit the mark, as I've not used most of these module formats outside of StealJS.

**Comment by matthewp** [comment]
Yep, looks good, thanks!

**Comment by m-mujica** [comment]
I think this PR would be way more awesome if it were a single atomic change (single commit) and speaking about commits, using a guideline like https://github.com/erlang/otp/wiki/Writing-good-commit-messages would make it even more amazing.

We don't have an official commit message guideline, but most of them look as nice as this one 

https://github.com/stealjs/steal/commit/f27647691fd862c312847ce069db5ec3e7604dad

just sayin....

**Comment by m-mujica** [comment]
Also, I think you need to rebase against master @leoj3n, not quite sure what's going on with Saucelabs. I've tried rebuilding your PR a couple of times and got the same errors.

**Comment by leoj3n** [comment]
I just assumed you would squash on merge. You can do it through the web UI
even. I'll rebase on my end though.

**Comment by leoj3n** [comment]
Normally I force push to my PRs, but I got yelled at for that recently because there were unsresolved Q/As (although I'm still not 100% sure if that actually bumps out the Q/As or not).

**Comment by matthewp** [comment]
Moved this PR over to https://github.com/stealjs/steal/pull/1260 which is merged with master so tests should pass. Thanks for the contribution!

**Pull request closed without merging**